test/controllers/update/check.your.answers.controller.spec.ts

```diff
@@ -270,7 +270,7 @@ describe("CHECK YOUR ANSWERS controller", () => {
       expect(resp.text).toContain(UPDATE_CHECK_YOUR_ANSWERS_PAGE_BENEFICIAL_OWNER_STATEMENTS_CEASED_TITLE);
       expect(resp.text).toContain(CHANGE_LINK);
       expect(resp.text).toContain(CHANGE_LINK_ENTITY_NAME);
-      expect(resp.text).toContain(CHANGE_LINK_ENTITY_INCORPORATION_COUNTRY);
+      expect(resp.text).not.toContain(CHANGE_LINK_ENTITY_INCORPORATION_COUNTRY);
```

Comment:
Rather than checking it's not present, I think the unit test change would be to just delete this line and also the definition of the CHANGE_LINK_ENTITY_INCORPORATION_COUNTRY constant.

I know we've changed the implementation here but our tests don't exhaustively check that all the links that shouldn't be there aren't, so looks a bit odd now including this check but not others. Do you agree?